### PR TITLE
Continue  #158

### PR DESCRIPTION
**Pull request by amio**
A clone of #158 but reverted the deletion of package-lock.json.

### CODE REVIEW

File: libs/index.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ Advanced usage (for badge makers):
       ['latest stable release', '/github/release/babel/babel/stable'],
       ['latest tag', '/github/tag/micromatch/micromatch'],
       ['watchers', '/github/watchers/micromatch/micromatch'],
+      ['status', '/github/status/micromatch/micromatch'],
+      ['status', '/github/status/tunnckoCore/badgen-service/github-status'],
```
Comment (amio) [inline libs/index.md]: This is a "status branch" example, duplicated with "micromatch/micromatch/dev", and I guess you gonna delete it after PR merged? @tunnckoCore

File: libs/index.md [hunk 2]
```diff
@@ -70,6 +70,10 @@ Advanced usage (for badge makers):
...
+      ['status branch', '/github/status/micromatch/micromatch/dev'],
```
Comment (amio) [inline libs/index.md]: micromatch doesn't have a `dev` branch

File: libs/index.md (hunk 1 above)
Comment (tunnckoCore) [reply libs/index.md]: Was going to mention after i PR-ed, that i wont deleting the repo and branch.

File: libs/index.md (hunk 2 above)
Comment (tunnckoCore) [reply libs/index.md]: It have.. :D

**Comment by tunnckoCore** [comment]
Was going to mention after i PR-ed, that i wont deleting the repo and branch.
At least, until we found some pending one? It's cool to show it. It won't be problem for me to not deleting it.

File: libs/index.md (hunk 2 above)
Comment (amio) [reply libs/index.md]: Isn't the dev branch this https://github.com/micromatch/micromatch/tree/dev ?

File: libs/index.md (hunk 1 above)
Comment (amio) [reply libs/index.md]: Not sure what the status would be if I merged this one 😂

**Comment by amio** [comment]
@tunnckoCore Seems this one ([micromatch/micromatch/gh-pages](https://badgen-service-ovkqzadxac.now.sh/github/status/micromatch/micromatch/gh-pages)) is always pending

**Comment by tunnckoCore** [comment]
Oh, yea probably is not found ;d;d

> is always pending.

haha, strange, but it's not guaranteed, IMO. Hm.. how we can make pending status? That's what we should learn :D THen we can add such branch here ;p

**Comment by amio** [comment]
The states looks a bit weird, sometimes it shows check status (success, error), sometime it looks like merging status (pending).

But anyway, let's merge this.

**Pull request merged**